src/site/layouts/faq_multiple_q_a.drupal.liquid

```diff
@@ -47,6 +47,7 @@
             <!-- TOC -->
             {% if fieldTableOfContentsBoolean %}
               <section id="table-of-contents">
+                <button class="vads-u-visibility--screen-reader">Back to top</button>
```

Comment:
I think we'll need `type="button"` on these buttons for a11y

Reply:
done.